airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/SyncWorkflowImpl.java

```diff
@@ -27,7 +27,7 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(SyncWorkflowImpl.class);
   private static final String VERSION_LABEL = "sync-workflow";
-  private static final int CURRENT_VERSION = 1;
+  private static final int CURRENT_VERSION = 2;
```

Comment:
do we need to switch to 2 just for a blank space change? or is this changing for some other reason?

Reply:
Good point. Missed that now that the activity implementation is the only thing that differs (not the interface and not a child workflow)